Sources/Networking/HTTPClient/ETagManager.swift

```diff
@@ -150,6 +143,14 @@ private extension ETagManager {
         responseCode == .notModified
     }
 
+    func shouldUseETag(_ response: Response, withSignatureVerification: Bool) -> Bool {
+        switch response.verificationResult {
+        case .verified: return true
+        case .notRequested: return !withSignatureVerification
```

Comment:
In Android, what we do, is that, when we detect that the cached customer info has a `NOT_REQUESTED` verification result and signature verification is enabled (with either informational or enforced), we clear both the customer info cache and the etag cache. I believe we do the same in iOS. Shouldn't that mean this shouldn't happen?

Reply:
Oh well, I guess that would only work for the customer info endpoint... There could be edge cases where this might not be tru for other verified endpoints. Will implement this in Android 👍

Reply:
Done in https://github.com/RevenueCat/purchases-android/pull/1114